curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/655774d01daeeaa1978b99d5.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 655774d01daeeaa1978b99d5
+title: Step 14
+challengeType: 20
+dashedName: step-14
+---
+
+# --description--
+
+At the beginning, all the other nodes in the graph are considered to be at infinite distance from the source node, because the distance has not determined yet.
```

Comment:
```suggestion
At the beginning, all the other nodes in the graph are considered to be at infinite distance from the source node, because the distance has not been determined yet.
```